service/frontend/admin_handler.go

```diff
@@ -37,11 +37,12 @@ import (
 	commonpb "go.temporal.io/api/common/v1"
 	historypb "go.temporal.io/api/history/v1"
 	replicationpb "go.temporal.io/api/replication/v1"
-	commonspb "go.temporal.io/server/api/common/v1"
-	historyspb "go.temporal.io/server/api/history/v1"
 	"google.golang.org/grpc/metadata"
 	"google.golang.org/protobuf/types/known/timestamppb"
 
+	commonspb "go.temporal.io/server/api/common/v1"
+	historyspb "go.temporal.io/server/api/history/v1"
+
```

Comment:
Remove empty line and resort 3rd section.

Reply:
Or just don't touch this file at all.

Reply:
Need to fix my GoLand configuration again.